charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,182 @@
+# Default values for fluentbit-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Set this to containerd or crio if you want to collect CRI format logs
+containerRuntime: docker
+# You can also deploy the Kubernetes logging stack with Helm, just need to set the Kubernetes parameter to ture.
+# see https://github.com/fluent/fluent-operator/tree/master/manifests/logging-stack
+Kubernetes: false
+
+operator:
+# The initial container is to get the actual storage path of the docker log files so that it can be mounted to collect the logs.
+# see https://github.com/fluent/fluent-operator/blob/master/manifests/setup/fluent-operator-deployment.yaml#L26
+  initcontainer:
+    repository: "docker"
+    tag: "20.10"
+  container:
+    repository: "kubesphere/fluent-operator"
+    tag: "v1.7.0"
+    # FluentBit operator resources. Usually user needn't to adjust these.
+  resources:
+    limits:
+      cpu: 100m
+      memory: 60Mi
+    requests:
+      cpu: 100m
+      memory: 20Mi
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  # For non-docker environments, the mount path for fluentbit uses a hard-coded path.
```

Comment:
Oh, you mean when not using Docker as a container runtime (which I think will be increasingly the norm) it uses this path (which I think is part of the K8S standard to be honest).

Maybe:
```suggestion
# The operator currently assumes a Docker container runtime path for the logs as the default, for other container runtimes you can set the location explicitly below.
```

It might be better just to have a single override as this chart does not allow mixed container runtimes anyway (e.g. nodes with Docker, CRIO and containerd in the same cluster). Rather than having to say use `crio` but maybe they don't configure it.

Reply:
> It might be better just to have a single override as this chart does not allow mixed container runtimes anyway (e.g. nodes with Docker, CRIO and containerd in the same cluster). Rather than having to say use `crio` but maybe they don't configure it.

Agree